crates/ruff_python_ast/src/visitor.rs

```diff
@@ -53,7 +53,7 @@ pub trait Visitor<'a> {
         walk_except_handler(self, except_handler);
     }
     fn visit_format_spec(&mut self, format_spec: &'a Expr) {
-        walk_expr(self, format_spec);
+        self.visit_expr(format_spec);
```

Comment:
I know it's sort of dumb / tedious, but wondering if we should instead add `walk_format_spec` that just calls `visitor.visit_expr(format_spec)`, for consistency with the rest of the trait.

Reply:
I can see the argument for that. Added in 125f33c34c340ac8d42196cfa674701711050551.

Reply:
Thanks. I think it will be clearer coming back to this code in the future, rather than having to ask myself, "Why does this use `self` unlike all the others?"